rs/replicated_state/src/canister_state/system_state/call_context_manager.rs

```diff
@@ -276,7 +276,7 @@ impl CallContextManagerStats {
 
     /// Calculates the stats for the given call contexts and callbacks.
     ///
-    /// Time complexity: `O(N)`.
+    /// Time complexity: `O(n)`.
```

Comment:
Nit: I believe that `O(n + m)` or `O(| call_contexts | + | callbacks |)` will be more accurate here.

Reply:
Switched to `O(|call_contexts| + |callbacks|)`.

The intent was simply to clarify that this was a linear time function. And this is mostly in contrast to `O(1)` functions, which are safe to call e.g. from the scheduler when aggreating metrics or checking invariants at runtime.